AdSecGH/Parameters/AdSecRebarLayerGoo.cs

```diff
@@ -36,9 +36,10 @@ public override string ToString() {
       } catch (Exception) {
         try {
           var byBarPitch = (ILayerByBarPitch)Value;
-          Length spacing = byBarPitch.Pitch.ToUnit(DefaultUnits.LengthUnitGeometry);
+          var spacing = byBarPitch.Pitch.ToUnit(DefaultUnits.LengthUnitGeometry);
           str = $"{bar} bars / {spacing}";
         } catch (Exception) {
+          /* don't expect to fail */
```

Comment:
why are we using Try/Catch then? :D Is it meant to continue without failing you mean? Can't we simply check if the failure lies to the fact that the above catch failed only?